docs/knowledgebase/integrate/subkey.md

```diff
@@ -15,13 +15,13 @@ transactions.
 ### Build from Source
 
 The Subkey binary, `subkey`, is also installed along with
-[Substrate](../getting-started/#fast-installation). If
-you want to play with just Subkey (and not Substrate), you will need to have the Substrate
-dependencies. Use the following two commands to install the dependencies and Subkey, respectively:
+[Substrate](../getting-started/#fast-installation). If you want to play with just Subkey (and not
+Substrate), you will need to have the Substrate dependencies. Use the following two commands to
+install the dependencies and Subkey, respectively:
 
 ```bash
 $ curl https://getsubstrate.io -sSf | bash -s -- --fast
-$ cargo install --force --git https://github.com/paritytech/substrate subkey
+$ cargo install --force subkey --git https://github.com/paritytech/substrate --tag v2.0.0-rc6
```

Comment:
Does the bug we talked 1 week ago was in `rc5` or `rc6`?
cc: @JoshOrndorff

Reply:
This is a good change, and will prevent exactly the kind of confusion Dan and Joe experienced above.

I'm not sure what bug you mean though.

Reply:
Exactly what @joepetrowski commented